monai/handlers/mlflow_handler.py

```diff
@@ -303,7 +303,9 @@ def _log_metrics(self, metrics: dict[str, Any], step: int | None = None) -> None
 
         run_id = self.cur_run.info.run_id
         timestamp = int(time.time() * 1000)
-        metrics_arr = [mlflow.entities.Metric(key, value, timestamp, step or 0) for key, value in metrics.items()]
+        metrics_arr = [
```

Comment:
Could you please also update the MLflow test case to cover this issue?

Thanks,  
Bin

Reply:
Added, please help take a look at it again, thanks.